### DISCUSSION TITLE
are there established coding style?

### DISCUSSION THREAD

**Discussion by scarf005** [pkgxdev/pkgx#680] [Help]
https://github.com/teaxyz/lib/blob/81bcf085d9dadb5bff4fbdde646ee32fe2c1fc88/deno.json#L6-L12
https://github.com/teaxyz/cli/pull/234

`fmt` section being intentionally disabled lead to me wondering if there's an established coding convention new contributors should follow. Tho judging by the source code, I guess this could go along well with the team's current style:

```jsonc
	"fmt": {
		"semiColons": false,
		"lineWidth": 120
	}
```

**Comment by jhheider** [comment]
It'd be great to get the coding style configured. mxcl is fond of saying its whatever he likes at the moment, which is reasonably true, but doesn't tend to drift much. I wonder if there's a tool to backfill style choices from an existing body of work...

**Comment by mxcl** [comment]
I'd be fine with a `fmt` configuration that didn’t require too many changes. We're pretty consistent at this point and it's time for me to stop imposing my craziness 😆
